stdlib/array.ml

```diff
@@ -30,6 +30,12 @@ external unsafe_blit :
 external create_float: int -> float array = "caml_make_float_vect"
 let make_float = create_float
 
+module Floatarray = struct
+  external create : int -> floatarray = "caml_floatarray_create"
+  external get : floatarray -> int -> float = "caml_floatarray_get"
+  external set : floatarray -> int -> float -> unit = "caml_floatarray_set"
+end
```

Comment:
Right. That's the user-facing part that I want to do in another PR because it will take longer to discuss and it's not really needed for benchmarking.